ibis/backends/base/sql/registry/string.py

```diff
@@ -80,3 +80,21 @@ def parse_url(translator, expr):
         return "parse_url({}, '{}', {})".format(
             arg_formatted, extract, key_fmt
         )
+
+
+def startswith(translator, expr):
+    arg, start = expr.op().args
+
+    arg_formatted = translator.translate(arg)
+    start_formatted = translator.translate(start)
+
+    return f"starts_with({arg_formatted}, {start_formatted})"
```

Comment:
I don't think any backend other than bigquery supports this, use `LIKE 'XXX%'` instead, which should work on all them.